skfem/element/element_tri/element_tri_p2b.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+from ..element_h1 import ElementH1
+from ...refdom import RefTri
+
+
+class ElementTriP2B(ElementH1):
+    """Quadratic triangular element with a cubic bubble."""
```

Comment:
I'm wondering if this is the same as [`element_tri_ccr`](https://github.com/kinnala/scikit-fem/blob/master/skfem/element/element_tri/element_tri_ccr.py), is it ?

Reply:
Thanks, yes. I deleted element_tri_ccr and also renamed element_tri_mini to element_tri_p1b. Those old names are still exported in __init__.